### ISSUE TITLE
Loading untrusted bootchain

### ISSUE THREAD

**Issue by sen0rxol0** [0x7ff/gaster#33]
It appears `gaster load` is not working properly on A9 devices.
Neither does `gaster pwn`,  possibly the fix is to clear usb handle:
https://github.com/0x7ff/gaster/blob/7ffffffb97c8fe0f1701fc9fa306149b75f3c493/gaster.c#L1893
the issue with loading images might be the 'go' command: 
https://github.com/0x7ff/gaster/blob/7ffffffb97c8fe0f1701fc9fa306149b75f3c493/gaster.c#L1904

**Comment by sen0rxol0** [comment]
Update: 
`gaster pwn` works fine actually.
Fix #33 does not solve loading though, ah it stops at waiting usb handle.
```
Now you can boot untrusted images.
[IOKit] Waiting for the USB handle with VID: ***, PID: ***
Found the USB handle.
[IOKit] Waiting for the USB handle with VID: ***, PID: ***
Found the USB handle.
[IOKit] Waiting for the USB handle with VID: ***, PID: ***
```
That means before the 'go' command, here:
https://github.com/0x7ff/gaster/blob/7ffffffb97c8fe0f1701fc9fa306149b75f3c493/gaster.c#L1903

**Comment by P5-2005** [comment]
> Update: 
> 
> `gaster pwn` works fine actually.
> 
> Fix #33 does not solve loading though, ah it stops at waiting usb handle.
> 
> ```
> 
> Now you can boot untrusted images.
> 
> [IOKit] Waiting for the USB handle with VID: ***, PID: ***
> 
> Found the USB handle.
> 
> [IOKit] Waiting for the USB handle with VID: ***, PID: ***
> 
> Found the USB handle.
> 
> [IOKit] Waiting for the USB handle with VID: ***, PID: ***
> 
> ```
> 
> That means before the 'go' command, here:
> 
> https://github.com/0x7ff/gaster/blob/7ffffffb97c8fe0f1701fc9fa306149b75f3c493/gaster.c#L1903

which ios you are trying?! If ios16 is not gaster issue at all, need update libirecovery to fix go/bootx issue

**Comment by 0x7ff** [comment]
Use irecovery for loading images, I removed the load command from gaster.
